storyblok/components.json

```diff
@@ -55,7 +55,9 @@
           "type": "bloks",
```

Comment:
we should run prettier on this file :)

Reply:
I saw that i messed up with this file and commited removed blocks, will fix 🙈